cl/recap/mergers.py

```diff
@@ -1399,3 +1405,56 @@ def process_orphan_documents(
             # exceptions that were previously raised for the
             # processing queue items a second time.
             pass
+
+
+def add_att_report_for_recap_email(
```

Comment:
This is copying a lot of code from elsewhere. Do you think we could generalize the other functions instead of duplicating them?

Reply:
Yeah, I've removed this method.

Now after getting the attachment page, we create a PQ using `make_attachment_pq_object` to process the attachment page, I tweaked that method to also accept the HTML attachment page (in case we get it from the notification URL, without querying PACER).

And then we use `process_recap_attachment` to merge the attached documents, I tweaked the method because here we need to return the attachments that were added so we can try to download them.

I tried to call these tasks asynchronously using a chain but I found some problems because we need to send the alerts and webhooks after we'd finished getting all the documents, and using the chain wasn't working well.  I think this might be possible using a combination of celery chains, groups, and chords, but it might complicate things a bit.